lib/net/http.rb

```diff
@@ -497,22 +517,28 @@ def HTTP.post(url, data, header = nil)
       }
     end
 
-    # Posts HTML form data to the specified URI object.
-    # The form data must be provided as a Hash mapping from String to String.
-    # Example:
+    # :call-seq:
+    #   Net::HTTP.post_form(uri, data) -> response
```

Comment:
This call-seq also does not provide much more info than the method definition.

Reply:
Removed.